src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx

```diff
@@ -105,6 +105,7 @@ class ActionBarActionsButtonDrawerOption extends React.PureComponent {
         return (
             <div
                 className={containerClasses}
+                data-testid="actions_button_drawer--option"
```

Comment:
Is it okay if these are not unique?
Also, do we need them?  I'm given to understand that Testing Library encourages you to try and find elements by visible text or role and such, and only use these as an absolute last resort.
See: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text

Reply:
Thank you for making me understand using `data-testid` isn't a great choice. So, I removed it. Fixed.

Reply:
All good!  We're all learning about this together!
It's always there as a "last resort" if there's no good way to find your thing using `*ByText()` or `*ByRole()` and the like.  It just shouldn't be the first choice, whereas I think we _were_ using it as the first choice in much of our earlier wave of testing with Testing Library, out of ignorance.